src/pages/components/index.mdx

```diff
@@ -1,12 +1,13 @@
 ---
 title: Overview
 description: >
-  A standard overview
+  Development instructions for building effective user interfaces.
 ---
 
-## Introduction
+import { OverviewLinks } from '../../components/MarkdownProvider';
```

Comment:
I am seeing some new warnings both locally and in the CI build 👀 

<img width="775" alt="Screen Shot 2020-06-15 at 11 45 36 AM" src="https://user-images.githubusercontent.com/1811365/84695098-28a4bd80-aeff-11ea-8721-5679dfe3c8a9.png">

I think these imports may not be needed, and once removed seems to address the warning logs.

Reply:
TY 👍